.gitignore

```diff
@@ -3,6 +3,7 @@
 *.sqlite
 *.sublime*
 *\.*swp
+.idea
```

Comment:
Gotcha.  Here's my suggestion:

1. collab supports a lot of apps, not just idea-box. We probably should add a gitignore entry for all the apps collab supports (check out https://github.com/cfpb/collab/blob/master/collab/urls.py for a list).
1. Since this isn't related to the actual change in this PR, I suggest reverting the idea change in this PR and performing step 1 in a separate PR.
1. Lastly, i think the list of apps in the gitignore from step 1 would be confusing unless they have a comment above them along the lines of `#This is a convenience for jetbrains IDE`